spec/generator/site_setting_rename_migrations_generator_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "rails/generators"
+require "generators/site_setting_rename_migration/site_setting_rename_migration_generator"
+
+RSpec.describe SiteSettingRenameMigrationGenerator, type: :generator do
+  it "generates the correct migration" do
+    freeze_time DateTime.parse("2010-01-01 12:00")
+    described_class.start(%w[site_description contact_email], destination_root: "#{Rails.root}/tmp")
+    file_path = "#{Rails.root}/tmp/db/migrate/20100101120000_rename_site_description_setting.rb"
+
+    expected_content = <<-EXPECTED_CONTENT
```

Comment:
Are we able to use the `<<~` squiggly heredoc to preserve the indentation of the content in this file?

Reply:
Good catch, thank you :)